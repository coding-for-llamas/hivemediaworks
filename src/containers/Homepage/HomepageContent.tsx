const background = 'https://www.dl.dropboxusercontent.com/s/9024lrtybjgx01f/background.jpg?dl=0';

const HomeContent = (): JSX.Element => (
  <>
      <img src={background} alt="Laptop" className="bg" />
      <div className="home-header">
        <h1 className="lg-heading">
          Rhayven
          {' '}
          <span className="text-secondary">Ayers</span>
        </h1>
        <h2 className="sm-heading">
          Accessible Web Developer, Programmer &amp; Designer
        </h2>
        <div className="icons">
          <a href="https://twitter.com/codingforllamas" aria-label="Twitter Page">
            <i className="fab fa-twitter fa-2x" />
          </a>
          <a href="https://www.facebook.com/codingforllamas" aria-label="Facebook Page">
            <i className="fab fa-facebook fa-2x" />
          </a>
          <a href="https://www.linkedin.com/in/rhayvenayers" aria-label="LinkedIn Page">
            <i className="fab fa-linkedin fa-2x" />
          </a>
          <a href="https://github.com/Unesdala" aria-label="Github Page">
            <i className="fab fa-github fa-2x" />
          </a>
        </div>
      </div>
  </>
);

export default HomeContent;
