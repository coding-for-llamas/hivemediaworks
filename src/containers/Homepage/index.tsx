import HomeContent from './HomepageContent';

export const Homepage = (): JSX.Element => {
  document.title = 'Coding For Llamas';  
  return (
      <div className="Site-content">
        <main className="home-main">
          <HomeContent />
        </main>
      </div>
  );
};

export default Homepage;
