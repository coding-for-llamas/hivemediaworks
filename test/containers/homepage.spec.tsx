/* eslint-disable @typescript-eslint/no-explicit-any */

import { shallow } from 'enzyme';
import { Homepage } from 'src/containers/Homepage/index';
import HomepageContent from 'src/containers/Homepage/HomepageContent';

describe('Homepage', () => {
  const wrapper = shallow(<Homepage />);
  it('renders', () => {
    expect(wrapper.find(HomepageContent).exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find(HomepageContent).dive().find('div.home-header').exists()).toBe(true);
  });
  
});
