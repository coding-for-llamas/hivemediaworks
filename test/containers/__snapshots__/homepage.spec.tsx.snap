// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage renders 1`] = `ShallowWrapper {}`;
